src/pages/profile/profile.js

```diff
@@ -20,15 +26,22 @@ function Profile() {
         }
     }
     async function onAddUser() {
+        if (firstName.length < 1 && lastName.length < 1 && email.length < 1) {
```

Comment:
this condition is only met when ALL fields are empty. Consider what happens, for example, when firstName is filled in, but lastName and email aren't.

You should have separate checks (and error handling) for every case:
- firstName is empty
- lastName is empty
- email is empty

Reply:
also, don't forget to reset error handling when a user updates the field.